src/main/java/it/gov/pagopa/payhub/ionotification/dto/mapper/IOServiceMapper.java

```diff
@@ -22,4 +23,18 @@ public IOService apply(String enteId, String tipoDovutoId, ServiceRequestDTO ser
                 .build();
 
     }
+
+    public ServiceDTO mapService(IOService service){
+        return ServiceDTO.builder()
+                .serviceId(service.getServiceId() != null ? service.getServiceId() : null)
+                .serviceName(service.getServiceName())
+                .status(service.getStatus())
+                .serviceDescription(service.getServiceDescription())
+                .organizationName(service.getOrganizationName())
+                .organizationDepartmentName(service.getOrganizationDepartmentName())
+                .organizationFiscalCode(service.getOrganizationFiscalCode())
+                .creationRequestDate(String.valueOf(service.getCreationRequestDate()))
+                .creationServiceDate(service.getCreationServiceDate() != null ? String.valueOf(service.getCreationServiceDate()) : null)
```

Comment:
String.valueOf is not the way to serialize a date! You have to use DateTimeFormatter or the .format method